### PR TITLE
Include swagger examples from any.example

### LINKED ISSUE #34: Feature: Include examples with any.example()
This is a feature request. =)

Joi offers to define some examples for any value.
Swagger allows kind of the [same](https://swagger.io/docs/specification/describing-request-body/) [thing](https://swagger.io/docs/specification/2-0/adding-examples/).

It would be nice to be able to define examples directly in Joi and have them show up in the generated swagger defintion.

### PR DESCRIPTION
**Pull request by Siilwyn**
Closes #34 

Hi @Twipped hope you find time to review this! :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,7 +104,10 @@ J2S returns a result object containing `swagger` and `components` properties. `s
 
 - `joi.alternatives()` defines the structure using the first schema provided on `.items()` (see below for how to override)
 
-- `any.default()` sets the `"default"` detail.
+- `joi.any()`
+  - `.default(5)` -> `"default": 5`
+  - `.example('hi')` -> `"example": "hi"`
+  - `.example('hi').example('hey')` -> `"examples": ["hi", "hey"]`
```
Comment (Twipped) [inline README.md]: You removed the documentation for any.default and replaced it with example code? This doesn't fit with the style of the existing documentation.

Comment (Twipped) [reply README.md]: Ah, I see where you were going. Yeah, for type specific stuff it gets indented below, but `any` isn't a type 😀  So the correct format would be to return `default` back to how it was, and then add a `any.example()` line with a short description.

Comment (Siilwyn) [reply README.md]: Oooh, gotcha!

Comment (Twipped) [reply README.md]: Perfect, thanks!

**Pull request merged**